src/traits/trait-objects.md

```diff
@@ -47,32 +39,40 @@ Memory layout after allocating `pets`:
 :   +-----------+-------+   :     :   +-----+-----+                             :
 :   | ptr       |   o---+---+-----+-->| o o | o o |                             :
 :   | len       |     2 |   :     :   +-|-|-+-|-|-+                             :
-:   | capacity  |     2 |   :     :     | |   | |   +---------------+           :
-:   +-----------+-------+   :     :     | |   | '-->| name: "Fido"  |           :
-:                           :     :     | |   |     +---------------+           :
+:   | capacity  |     2 |   :     :     | |   | |   +-------------+             :
+:   +-----------+-------+   :     :     | |   | '-->| name: Fido  |             :
+:                           :     :     | |   |     +-------------+             :
 `- - - - - - - - - - - - - -'     :     | |   |                                 :
-                                  :     | |   |     +----------------------+    :   
-                                  :     | |   '---->| "<Dog as Pet>::name" |    :
-                                  :     | |         +----------------------+    : 
-                                  :     | |                                     : 
-                                  :     | |   +-+                               :   
-                                  :     | '-->|\|                               :     
-                                  :     |     +-+                               :    
-                                  :     |                                       : 
-                                  :     |     +----------------------+          : 
-                                  :     '---->| "<Cat as Pet>::name" |          : 
+                                  :     | |   |     +----------------------+    :
```

Comment:
Could these be something like
```
vtable:
  talk: Dog::talk
```

Reply:
Yeah, let me try that!

Reply:
I added small labels to the boxes, I hope you like it.